helm/elixier/templates/spark3/_helpers/spark-defaults.conf.tpl

```diff
@@ -39,7 +39,7 @@ spark.blockManager.port {{ .Values.spark.blockmanager_port | default "2044" }}
 spark.executor.cores {{ .Values.spark.executor_cores }}
 spark.executor.memory {{ .Values.spark.executor_memory }}
 
-
+spark.driver.host ${env:HOSTNAME}
```

Comment:
Remove this and see if stuff still works or not

Reply:
Removed this commit too, in addition with this commit too [https://github.com/elixierdata/helm/pull/1/files/377e5c91dfa962cd8fec085e2544adf6feb9e461#r1068088943](https://github.com/elixierdata/helm/pull/1/files/377e5c91dfa962cd8fec085e2544adf6feb9e461#r1068088943)

Jupyter & spark are working